### PR TITLE
Update bonita-purpose.adoc

### PR DESCRIPTION
**Pull request by NathalieC**
* remove bonita architecture overview

### CODE REVIEW

File: modules/ROOT/pages/bonita-purpose.adoc [hunk 1]
```diff
@@ -1,86 +1,37 @@
...
-Bonita integrates with existing information systems, orchestrates heterogeneous systems, and provides deep visibility into processes across the organization.
+{description}
 
-Bonita Platform has two components: the development environment, Bonita Studio, and the runtime environment, Bonita Runtime.
+== Values of Bonita Platform
 
-== What can I do with Bonita?
+As a platform, Bonita supports your automation project from implementation to maintenance and evolution, with a set of dedicated tools. +
+It comes with plenty of provided artifacts, speeding up the development, facilitating the communication among team members, but also easing the maintenance. +
+As an open platform, it encourages the creation of more artifacts to perfectly fit the business and technical needs of a given automation project. As our customers say; "you are never stuck with Bonita". +
```
Comment (alachambre) [inline modules/ROOT/pages/bonita-purpose.adoc:10]: lol

File: modules/ROOT/pages/bonita-purpose.adoc [hunk 2]
```diff
@@ -1,86 +1,37 @@
...
+Plus, its robustness enables the BPM engine to handle a company's critical projects, with high performance and technical monitoring indicators.
 
-Bonita is a Digital Process Automation (DPA) platform that allows you to create applications based on processes. Processes can be partially automated (if involving users) or fully automated.
+== Types of automation projects
 
-DPA, available with all Bonita editions, is applicable to a wide range of use cases: applications can go from loan request management, to student application processes, to analytical processes in drug discovery - and more! You can find the various industries where Bonita is used as well as use cases from our https://www.bonitasoft.com/industries[website dedicated page].
+=== The need for automation
+Some business processes do not need to be automated. Their execution stays in the scope of one team, the data they handle does not need to be retrieved nor sent from/to many places, nor undergo transformations from one step to the other, the execution of each instance does not need traceability nor reminders, and there is no need of reporting about their execution. +
```
Comment (alachambre) [inline modules/ROOT/pages/bonita-purpose.adoc]: This sentence is way to long, in a fully negation style, hard to follow IMHO. I suggest to split it.

File: modules/ROOT/pages/bonita-studio.adoc [hunk 3]
```diff
@@ -5,9 +5,17 @@ Discover what Bonita Studio is, as well as its main features.
 
 == What Bonita Studio is
 
-Bonita Studio is the central tool to design processes and applications. This is the place to:
+Bonita Studio is the central tool to design processes, data and applications. +
```
Comment (alachambre) [inline modules/ROOT/pages/bonita-studio.adoc]:
```suggestion
Bonita Studio is the central tool to design processes, data models and applications. +
```

File: modules/ROOT/pages/bonita-studio.adoc [hunk 4]
```diff
@@ -5,9 +5,17 @@ Discover what Bonita Studio is, as well as its main features.
...
+It is suited for both citizen developers (business analysts, project managers, power users) and application developers, with back-end java developers working in Bonita Studio and collaborating with front-end developers working in the UI Designer. +
 
-* xref:diagram-overview.adoc[Draw processes]
+Bonita Studio contains a *Bonita Runtime* (Tomcat, UI Designer, Bonita Portal, Bonita Engine, and an h2 database), suitable for testing an application that is in development. + When you run a process, it is automatically deployed onto the development platform. +
```
Comment (alachambre) [inline modules/ROOT/pages/bonita-studio.adoc]:
```suggestion
Bonita Studio contains a *Bonita Runtime* (Tomcat, UI Designer, Bonita Portal, Bonita Engine, and an h2 database), suitable for testing an application that is in development. +
When you run a process, it is automatically deployed onto the development platform. +
```

File: modules/ROOT/pages/what-is-bonita-index.adoc [hunk 5]
```diff
@@ -1,10 +1,35 @@
...
-:description: A useful overview of our purpose, the automation project artifacts, and the Bonita Platform architecture.
+:description: Bonita is an open-source and extensible platform for business process automation and optimization.
 
-A useful overview of Bonita Platform's purpose, the concepts used throughout the platform, and its high-level architecture.
+{description}
+
+== Goal
+*Bonita Platform accelerates the development, go-to-production, and maintenance of automation projects.* +
+While allowing users to execute tasks that impact their business data, it also efficiently *integrates with existing information systems* and *orchestrates heterogeneous systems*, some of them being https://www.bonitasoft.com/robotic-process-automation[soft robots]. +
+It provides *deep visibility of process execution* across the organization through its provided end-user applications but also, when needed, the xref:applications.adoc[Living applications] the project team can build to perfectly fit the business needs.
```
Comment (alachambre) [inline modules/ROOT/pages/what-is-bonita-index.adoc]:
```suggestion
It provides *deep visibility of process execution* across the organization through its ebbeded end-user applications or the xref:applications.adoc[Living applications] built by the project team to perfectly fit the business needs. 
```

File: modules/ROOT/pages/what-is-bonita-index.adoc [hunk 6]
```diff
@@ -1,10 +1,35 @@
...
+== Structure
+Bonita Platform is made of three main components: 
+
+* xref:bonita-studio.adoc[*Bonita Studio*] for the development phase of the projects
+* xref:bonita-bpm-overview.adoc#platform[*Bonita Runtime*] for the various runtime phases, running the BPM engine and the applications
+* https://documentation.bonitasoft.com/bcd/latest/[*Bonita Continuous Delivery*] to ease the iterative deployment of the projects from environment to environment +
+
+== Teamwork
+Bonita Studio offers a separation between visual coding capabilities and Java/web coding capabilities, to enable an efficient teamwork between business and technical teams. +
+While the citizen developers input the business aspects of the projects (process, data, main aspects of the user interfaces), the professional developers support the projects by taking care of the technical configuration and behavior of the business elements, and by coding what makes each project unique: they create what we call xref:managing-extension-studio.adoc[extensions] that they contribute to the projects as Maven dependencies. +
```
Comment (alachambre) [inline modules/ROOT/pages/what-is-bonita-index.adoc]:
```suggestion
While the citizen developers input the business aspects of the projects (process, data model, main aspects of the user interfaces), the professional developers support the projects by taking care of the technical configuration and behavior of the business elements, and by coding what makes each project unique: they create what we call xref:managing-extension-studio.adoc[extensions] that they contribute to the projects as Maven dependencies. +
```

File: modules/ROOT/pages/what-is-bonita-index.adoc [hunk 7]
```diff
@@ -1,10 +1,35 @@
...
+
+Both parts of this unified project team use BPMN (Business Process Model and Notation) as their common language, and can rely on the xref:project-documentation-generation.adoc[project documentation] to share the projects vision, needs, and achievements. +
+
+== Editions
+With the Open Source edition *Community*, experiment the value of the platform, create, and operate pretty complex projects. +
+For core and critical projects, for enterprise-grade performance and for support, choose our edition per subscription: the https://www.bonitasoft.com/pricing[*Enterprise* edition]. +
```
Comment (alachambre) [inline modules/ROOT/pages/what-is-bonita-index.adoc]: Mixing bold and links doesn't render well. 
```suggestion
For core and critical projects, for enterprise-grade performance and for support, choose our edition per subscription: the https://www.bonitasoft.com/pricing[Enterprise edition]. +
```

**Review by alachambre — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/bonita-purpose.adoc (hunk 2 above)
Comment (NathalieC) [reply modules/ROOT/pages/bonita-purpose.adoc]: ok. Working on it.

File: modules/ROOT/pages/what-is-bonita-index.adoc (hunk 5 above)
Comment (NathalieC) [reply modules/ROOT/pages/what-is-bonita-index.adoc]: fixed the typo in "embed" and committed

File: modules/ROOT/pages/bonita-purpose.adoc [hunk 8]
```diff
@@ -1,86 +1,45 @@
...
+=== The business applications types 
+This is where business analysts need to take one step aside and look at the overall way the end-users work. +
 
-Bonita Studio contains all the elements needed to develop and compile a Bonita application. It is a graphical environment for creating processes, applications, data models, and users views (pages and forms). It contains three major design tools:
+* Their job may require to be process-centric and do tasks from different processes as they appear in their todo list, with some filtering options. The task-list application is what you need. As an option, team coordinators can pre-sort and assign tasks as they become available, let the team organize themselves, or have the assignment criteria implemented in the process xref:actors.adoc[actor mapping]. Bonita provides such a task list in the xref:user-application-overview.adoc[User application].
+* Their job may require to be data-centric, with very standardized processes creating, modifying and sharing data in a consisteny way. In this case, a BPM approach with a Living application displaying lists of cases (requests, claims, invoices, tickets,...) to handle is the right way to go.
```
Comment (florentsarat) [inline modules/ROOT/pages/bonita-purpose.adoc]:
```suggestion
* Their job may require to be data-centric, with very standardized processes creating, modifying and sharing data in a consistent way. In this case, a BPM approach with a Living application displaying lists of cases (requests, claims, invoices, tickets,...) to handle is the right way to go.
```

File: modules/ROOT/pages/bonita-purpose.adoc [hunk 9]
```diff
@@ -1,86 +1,45 @@
...
 
-Bonita is a Digital Process Automation (DPA) platform that allows you to create applications based on processes. Processes can be partially automated (if involving users) or fully automated.
+== Types of automation projects
 
-DPA, available with all Bonita editions, is applicable to a wide range of use cases: applications can go from loan request management, to student application processes, to analytical processes in drug discovery - and more! You can find the various industries where Bonita is used as well as use cases from our https://www.bonitasoft.com/industries[website dedicated page].
+=== The need for automation
 
-Starting with Bonita 7.10, you can also handle Adaptive Case Management (ACM) with Bonita.
-ACM is like DPA in the sense that it allows to handle cases with some automation. But it also relies on strong principles that make it different from DPA:
+Some business processes do not need to be automated. +
```
Comment (florentsarat) [inline modules/ROOT/pages/bonita-purpose.adoc]: Some business processes need to be automated before others ?

It is a bit better to explain that they should focus on critical process first to later on automate less criticals one than just saying that some process don't need to be automated.

File: modules/ROOT/pages/bonita-purpose.adoc [hunk 10]
```diff
@@ -1,86 +1,45 @@
...
+Indeed, the benfits of automation are huge when:
```
Comment (florentsarat) [inline modules/ROOT/pages/bonita-purpose.adoc]:
```suggestion
Indeed, the benefits of automation are huge when:
```

File: modules/ROOT/pages/bonita-studio.adoc [hunk 11]
```diff
@@ -5,9 +5,18 @@ Discover what Bonita Studio is, as well as its main features.
...
-* xref:diagram-overview.adoc[Draw processes]
+Bonita Studio contains a *Bonita Runtime* (Tomcat, UI Designer, Bonita Portal, Bonita Engine, and an h2 database), suitable for testing an application that is in development. +
+When you run a process, it is automatically deployed onto the development platform. +
+
+Bonita Studio is not intended for any use other than development. As a consequence, the Bonita stack embedded into the Bonita Studio can not be used for production purposes.
+
+
+== Main features
+
+* xref:diagram-overview.adoc[Draw processes] on the whiteboard
```
Comment (florentsarat) [inline modules/ROOT/pages/bonita-studio.adoc:19]: Confused on why the "on the whiteboard" detail is needed? (and in darkmode we have a darkboard ;) )

File: modules/ROOT/pages/what-is-bonita-index.adoc [hunk 12]
```diff
@@ -1,10 +1,35 @@
...
+* xref:bonita-bpm-overview.adoc#platform[*Bonita Runtime*] for the various runtime phases, running the BPM engine and the applications
+* https://documentation.bonitasoft.com/bcd/latest/[*Bonita Continuous Delivery*] to ease the iterative deployment of the projects from environment to environment +
+
+== Teamwork
+Bonita Studio offers a separation between visual coding capabilities and Java/web coding capabilities, to enable an efficient teamwork between business and technical teams. +
+While the citizen developers input the business aspects of the projects (process, data model, main aspects of the user interfaces), the professional developers support the projects by taking care of the technical configuration and behavior of the business elements, and by coding what makes each project unique: they create what we call xref:managing-extension-studio.adoc[extensions] that they contribute to the projects as Maven dependencies. +
+
+Both parts of this unified project team use BPMN (Business Process Model and Notation) as their common language, and can rely on the xref:project-documentation-generation.adoc[project documentation] to share the projects vision, needs, and achievements. +
```
Comment (florentsarat) [inline modules/ROOT/pages/what-is-bonita-index.adoc]:
```suggestion
Both parts of this unified project team use BPMN (Business Process Model and Notation) as their common language, and can rely on the xref:project-documentation-generation.adoc[project documentation] to share the project vision, needs, and achievements. +
```

File: modules/ROOT/pages/bonita-purpose.adoc (hunk 9 above)
Comment (NathalieC) [reply modules/ROOT/pages/bonita-purpose.adoc]: I understand what you mean. I will let @harissubasic comment on this. I took this sentence from a conversation with Haris.

File: modules/ROOT/pages/bonita-studio.adoc (hunk 11 above)
Comment (NathalieC) [reply modules/ROOT/pages/bonita-studio.adoc:19]: It is not needed. The whiteboard is presented in the LAD training. This is to bring consistency.
I can remove it if you @florentsarat think it is not appropriate (indeed, I had not thought of the dark mode).

File: modules/ROOT/pages/bonita-purpose.adoc (hunk 2 above)
Comment (NathalieC) [reply modules/ROOT/pages/bonita-purpose.adoc]: done

File: modules/ROOT/pages/bonita-studio.adoc (hunk 11 above)
Comment (florentsarat) [reply modules/ROOT/pages/bonita-studio.adoc:19]: Ok, if it is to match the training - I fine with that
